### ISSUE TITLE
Dont see a machine details after creating a machine on softserve

### ISSUE THREAD

**Issue by vh05** [gluster/project-infrastructure#114]
I think I do not have access to create VM on soft-serve. Please provide me the access

**Comment by deepshikhaaa** [comment]
Are you a member of gluster org on Github?

**Comment by vh05** [comment]
Yes. I have been a gluster org member but not sure Github? Once I had raised request for that and got access to add Labels etc.

Please verify once

**Comment by deepshikhaaa** [comment]
Can you again try with new values in the provisioning form of softserve. Please make sure that the auto-populated public key is valid or not.
